subsys/bluetooth/services/fast_pair/fp_storage/include/fp_storage_manager.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Last comment - I would move this to internal includes (this is not part of the fp_storage API)